Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM condaforge/mambaforge:latest
+
+# For opencontainers label definitions, see:
+#    https://github.com/opencontainers/image-spec/blob/master/annotations.md
+LABEL org.opencontainers.image.title="RAiDER"
+LABEL org.opencontainers.image.description="Raytracing Atmospheric Delay Estimation for RADAR"
+#LABEL org.opencontainers.image.vendor=""
+#LABEL org.opencontainers.image.authors=""
```

Comment:
@dbekaert anything you'd like placed here?

Reply:
What would you suggest in terms of authors for open-source packages. Main contributors; but how would it change over time?